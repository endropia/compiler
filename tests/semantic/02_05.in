var
	i: integer;
begin
	i := 0;
	while (i) do begin
	i += 1;
	end;
end.