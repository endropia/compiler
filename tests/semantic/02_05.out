(5, 9) Boolean expected